frontend/src/metabase/public/hooks/use-embed-frame-options.ts

```diff
@@ -1,18 +1,10 @@
 import type { Location } from "history";
-import { useEffect } from "react";
 
 import type { DashboardUrlHashOptions } from "metabase/dashboard/types";
 import { parseHashOptions } from "metabase/lib/browser";
 import { isWithinIframe } from "metabase/lib/dom";
-import { useDispatch } from "metabase/lib/redux";
-import { setInitialUrlOptions } from "metabase/redux/embed";
 
 export const useEmbedFrameOptions = ({ location }: { location: Location }) => {
-  const dispatch = useDispatch();
```

Comment:
I removed this part as `setInitialUrlOptions` is touching the `embed` slice which is only used by interactive embedding.

This hook is supposed to be used for static+public.

It's currently still used in DashboardControls (which is used on the normal dashboard too) but I'll try to address that in another PR